### DISCUSSION TITLE
HPC Azure Configuration Optimization

### DISCUSSION THREAD

**Discussion by fcurow** [firemodels/fds#12969] [General]
Hello FDS community,

I am running a 70 core simulation on an HPC Azure instance. It is running slower than my 15 core local machine. I may have picked a hardware option not well suited for running FDS. Does anyone know of an optimal configuration with HPC Azure and can share their experience?

**Comment by Er9y714** [comment]
Rather than being optimal, the CPU might have less performance per core than your local machine. If you can find the CPU information on Azure, you can check the benchmark performance scores and compare with your local machine.

**Reply by fcurow** [reply]
Thank you, I neglected to mention that the core speed for the 72 core machine speed is 2.59 GHz and the 15 core machine is a bit faster, but not enough to compensate.
